extensions/prost/private/prost.bzl

```diff
@@ -261,7 +261,10 @@ def _rust_prost_aspect_impl(target, ctx):
     # https://github.com/rust-analyzer/rust-analyzer/blob/2021-11-15/crates/project_model/src/workspace.rs#L529-L531
     cfgs = ["test", "debug_assertions"]
 
+    crate_id = "prost-" + dep_variant_info.crate_info.root.path
```

Comment:
We don't need an extra prefix, I put it there to avoid encoding/suggesting that this should be exactly the root path.
Happy to remove if you prefer.

(and because rust_analyzer.bzl uses a prefix presumably for the same reason)